databricks_cli/sdk/service.py

```diff
@@ -202,19 +202,17 @@ def get_job(self, job_id, headers=None, version=None):
         )
 
     def list_jobs(
-        self, job_type=None, expand_tasks=None, limit=None, offset=None, headers=None, version=None
+        self, job_type=None, offset=None, limit=None, expand_tasks=None, headers=None, version=None
```

Comment:
Order of these kwargs has changed. I doubt there are call sites where all arguments are specified sequentially and where this change proves to be incompatible.

Regardless, it justifies a minor bump.

Reply:
I saw people using databricks-cli as SDK, plus it's used in package like https://pypi.org/project/databricks-api/, so it could be a breaking change

Reply:
In the latest iteration of the PR all new arguments are appended to the list.